UFTBuildTask/AlmLabManagementTask/localTask.ps1

```diff
@@ -46,7 +45,7 @@ if ($rerunIdx) {
 }
 
 $CDA1 = [bool]($varUseCDA) 
-Invoke-AlmLabManagementTask $varAlmServ $varUserName $varPass $varDomain $varProject $varRunType $varTestSet $varDescription $varTimeslotDuration $varEnvironmentConfigurationID $varReportName $CDA1 $varDeploymentAction $varDeploymentEnvironmentName $varDeprovisioningAction $buildNumber -Verbose
```

Comment:
We should keep $varTunType since it is used in [InvokeAlmLabManagementTaskCmdlet.cs](https://github.com/MicroFocus/ADM-TFS-Extension/blob/bb5865262bc015730edbfe6c33c6bdb5ab49b4c9/PSModule/InvokeAlmLabManagementTaskCmdlet.cs).